### DISCUSSION TITLE
Documentation:  FMQs (baskets) being released by FDA

### DISCUSSION THREAD

**Discussion by millerg23** [pharmaverse/admiral#2502] [General]
### Please select a category the issue is focused on?

User Guides

### Let us know where something needs a refresh or put your idea here!

Should we add something to admiral to show how to create the FMQs, and add to the ADAE dataset. Please see https://www.regulations.gov/document/FDA-2022-N-1961-0001 and download file to see spreadsheet containing FMQs.

I would suggest creating a Vignette showing how to create an AAG VAD using `admiral` functions, and using the FMQs supplied by FDA as example custom baskets. We could store just a subset of the FMQs in `admiral` repo, and make it clear `admiral` is not storing the FMQs for use, and users should go to the FDA website, to download the latest FMQs if they need them.
Within the Vignette we could also show the calls required in `ADAE` domain to get the flags from `AAG` added to `ADAE` VAD

**Comment by millerg23** [comment]
@bms63 I added some more detail above about what we could do, let me know if this is sufficient.

**Comment by bundfussr** [comment]
Do we need to investigate if FMQs are still used/provided by FDA?

The Roche standard group wondered why the MedDRA version is not stated in the document and why there are no frequent updates of the FMQs for each new MedDRA version.

The FMQs are defined in a document labelled as workshop material. I couldn't find an official page regarding FMQs.

**Comment by bms63** [comment]
@pharmaverse/admiral and @pharmaverse/admiral_comm anyone got any leads on FMQs?

**Comment by nbrucken17** [comment]
FMQs are still being developed by the FDA- they have currently defined 104 FMQs, and expect at least 30 more in their next release.  There is some information on FMQs from January 2023 posted on the FDA website, at [https://www.fda.gov/media/164639/download](https://www.fda.gov/media/164639/download), though the draft version of the latest FMQs seems to have disappeared.  They did give an update on FMQs at DIA earlier this year, confirming that they are still under active development.

The ADaM team is also working on analysis dataset structures for incorporating FMQs.  The FMQ sub-team presented a poster at the 2023 CDISC Interchange on FMQs, which does not appear to be available online.  Please contact me if you'd like a copy, and I can put you in touch with the author, Guowei Wu.

Thanks,
    Nancy

**Comment by manciniedoardo** [comment]
@nbrucken17 thanks for the input - would be good to see the poster to see if there are any possibilities for {admiral} to provide guidance.

**Comment by bms63** [comment]
I feel like this is more of a discussion rather an issue.  going to move over there.

**Comment by bms63** [comment]
@millerg23 or @nbrucken17 any updates on the world of FMQs?

**Reply by manciniedoardo** [reply]
@millerg23 @nbrucken17 just bumping this again
